Assets/Mirror/Examples/Room/Scripts/PlayerScore.cs

```diff
@@ -12,7 +12,7 @@ public class PlayerScore : NetworkBehaviour
 
         public override void OnStartServer()
         {
-            index = connectionToClient.connectionId;
+            index = ((NetworkRoomManager)NetworkManager.singleton).clientIndex;
```

Comment:
can’t you use GetComponent<Player>.playerNo here?

Reply:
Removed redundant initializer